solutions/usaco-1061.mdx

```diff
@@ -106,4 +106,129 @@ for x in eat:
 
 </PySection>
 
+<JavaSection>
+
+```java
+
```

Comment:
```suggestion
```
forgot this lol

Reply:
@sathu-beep was [concerned](https://github.com/cpinitiative/usaco-guide/pull/2092#pullrequestreview-730140332) about GitHub file preview, and @ryanchou-dev said the [extra line ](https://github.com/cpinitiative/usaco-guide/pull/2092#discussion_r689015036 )was how to get it working. Should I sacrifice the file GitHub preview for the raw file formatting then?

Reply:
Ahh nvm; just saw the line number

Reply:
an extra line is needed for github preview between the tag and markdown (so between `<JavaSection>` and ` ```java`), but this is optional and not needed for the website

you don't need an extra line between the start of the codeblock and the code (i think the website will strip it out, but it's unnecessary), so between ` ```java` and your code